logcheck/pkg/logcheck.go

```diff
@@ -172,6 +175,15 @@ func checkForFunctionExpr(fexpr *ast.CallExpr, pass *analysis.Pass, c *config) {
 					checkForFormatSpecifier(fexpr, pass)
 				}
 			}
+			// verbose Zero Check
+			if c.isEnabled(verboseZeroCheck, filename) && checkForVerboseZero(selExpr.X, pass) {
+				msg := fmt.Sprintf("Logging with V(0) is semantically equivalent to the same expression without it and just causes unnecessary overhead. It should get removed.")
+				pass.Report(analysis.Diagnostic{
+					Pos:     fexpr.Fun.Pos(),
+					Message: msg,
+				})
+			}
```

Comment:
Can you move the `pass.Report` down into `checkForVerboseZero` to avoid the code duplication? If we ever decide to make it better (for example, by suggesting a code update), then we only need to make that change in one place.

`checkForVerboseZero` can call a `isVerboseZero` that is equivalent to the current function.

Reply:
Yeah, this part is repeated. 
I have optimized it.